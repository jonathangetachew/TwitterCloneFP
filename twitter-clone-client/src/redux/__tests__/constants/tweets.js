import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import tweetConstants from '../../constants/tweets';

describe('TweetsConstants', () => {
    jest.useFakeTimers();

    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it(`render constants`, () => {
        expect(tweetConstants).toMatchSnapshot();
    });
});