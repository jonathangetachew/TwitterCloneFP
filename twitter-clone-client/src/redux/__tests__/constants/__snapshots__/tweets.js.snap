// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TweetsConstants render constants 1`] = `
Object {
  "GET_TWEETS": "GET_TWEETS",
  "GET_TWEETS_FAIL": "GET_TWEETS_FAIL",
  "GET_TWEETS_REQUEST": "GET_TWEETS_REQUEST",
  "GET_TWEETS_SUCCESS": "GET_TWEETS_SUCCESS",
  "SEARCH_TWEETS": "SEARCH_TWEETS",
  "SEARCH_TWEETS_FAIL": "SEARCH_TWEETS_FAIL",
  "SEARCH_TWEETS_REQUEST": "SEARCH_TWEETS_REQUEST",
  "SEARCH_TWEETS_SUCCESS": "SEARCH_TWEETS_SUCCESS",
}
`;
